mobx/lib/home_page.dart

```diff
@@ -16,13 +16,13 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
```

Comment:
Should make this Stateless as well :-)

Reply:
@pavanpodila I have already made that change, I think this is an outdated change.